cluster/gce/config-default.sh

```diff
@@ -387,12 +387,12 @@ NETWORK_PROVIDER="${NETWORK_PROVIDER:-kubenet}" # none, kubenet
 # Network Policy plugin specific settings.
 NETWORK_POLICY_PROVIDER="${NETWORK_POLICY_PROVIDER:-none}" # calico
 
-NON_MASQUERADE_CIDR="0.0.0.0/0"
+export NON_MASQUERADE_CIDR="0.0.0.0/0"
```

Comment:
Double quotes superfluous since nothing is to be expanded.
Use either no quotes or single quotes '0.0.0.0/0'.